temba/channels/models.py

```diff
@@ -2014,7 +2014,8 @@ def send_vumi_message(cls, channel, msg, text):
         channel.config['transport_name'] = 'ussd_transport' if is_ussd else 'mtech_ng_smpp_transport'
 
         payload = dict(message_id=msg.id,
-                       in_reply_to=None,
+                       in_reply_to=(msg.response_to.external_id
+                                    if msg.response_to_id else None),
```

Comment:
Right, this now makes more sense to me. I was thinking the MsgStruct was a thing only used for testing to mock out a Message. I'll update.